packages/website/src/components/Playground.tsx

```diff
@@ -50,7 +50,7 @@ function rangeReducer<T extends SelectedRange | null>(
 }
 function Playground(): JSX.Element {
   const [state, setState] = useHashState({
-    jsx: false,
+    fileType: 'ts',
```

Comment:
I think we should default this to TSX.
IMO it's the rarer case that people want the non-TSX supported syntax (which really is just angle bracket assertions), and much more common that people want to use JSX in the playground.